hypertrace-core-graphql-log-event-schema/src/main/java/org/hypertrace/core/graphql/log/event/fetcher/LogEventFetcher.java

```diff
@@ -0,0 +1,44 @@
+package org.hypertrace.core.graphql.log.event.fetcher;
+
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import java.util.concurrent.CompletableFuture;
+import javax.inject.Inject;
+import org.hypertrace.core.graphql.atttributes.scopes.HypertraceCoreAttributeScopeString;
+import org.hypertrace.core.graphql.common.fetcher.InjectableDataFetcher;
+import org.hypertrace.core.graphql.common.request.ResultSetRequestBuilder;
+import org.hypertrace.core.graphql.log.event.dao.LogEventDao;
+import org.hypertrace.core.graphql.log.event.schema.LogEventResultSet;
+
+public class LogEventFetcher extends InjectableDataFetcher<LogEventResultSet> {
+
+  public LogEventFetcher() {
+    super(LogEventFetcherImpl.class);
+  }
+
+  static final class LogEventFetcherImpl implements
+      DataFetcher<CompletableFuture<LogEventResultSet>> {
+    private final ResultSetRequestBuilder requestBuilder;
+    private final LogEventDao logEventDao;
+
+    @Inject
+    LogEventFetcherImpl(ResultSetRequestBuilder requestBuilder, LogEventDao logEventDao) {
+      this.requestBuilder = requestBuilder;
+      this.logEventDao = logEventDao;
+    }
+
+    @Override
+    public CompletableFuture<LogEventResultSet> get(DataFetchingEnvironment environment) {
+      return this.requestBuilder
+          .build(
+              environment.getContext(),
+              HypertraceCoreAttributeScopeString.SPAN,
```

Comment:
```suggestion
              HypertraceCoreAttributeScopeString.LOG_EVENT,
```

Reply:
I have added a new LogEventRequestBuilder class (not sure if thats the way to go).